pulsar-broker/src/test/java/org/apache/pulsar/client/impl/ReaderTest.java

```diff
@@ -906,6 +906,49 @@ public void testHasMessageAvailableAfterSeekTimestamp(boolean initializeLastMess
         }
     }
 
+    @Test(dataProvider = "initializeLastMessageIdInBroker")
+    public void testHasMessageAvailableAfterSeekTimestampWithMessageIdInclusive(boolean initializeLastMessageIdInBroker)
```

Comment:
I don't really understand the purpose of `initializeLastMessageIdInBroker`. I tried looking at the existing tests that have it, but I don't see why it's used. Could it be removed from this test?